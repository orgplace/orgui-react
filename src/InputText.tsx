import classNames from 'classnames';
import React from 'react';

const OuiInputText: React.FC<React.InputHTMLAttributes<HTMLInputElement>> = (props) => {
  return (
    <input {...props} type="text" className={classNames(
      'ouiInputText',
      props.className,
    )} />
  );
}

export { OuiInputText };
