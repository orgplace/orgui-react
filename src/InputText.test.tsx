import React from 'react'
import { OuiInputText } from './InputText'
import renderer from 'react-test-renderer';


test('OuiInputText', function() {
	const component = renderer.create(<OuiInputText />);

	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
