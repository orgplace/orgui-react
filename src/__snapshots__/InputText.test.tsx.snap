// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OuiInputText 1`] = `
<input
  className="ouiInputText"
  type="text"
/>
`;
